.travis.yml

```diff
@@ -77,6 +77,7 @@ script:
 - ./gradlew test
 - ./gradlew assemble_testNet3Debug
 - rm -rf .gradle/4.4
+- openssl aes-256-cbc -K $encrypted_1615d4cf65a2_key -iv $encrypted_1615d4cf65a2_iv -in .deploy/UpholdConstants.enc -out uphold-integration/src/main/java/org/dash/wallet/integration/uphold/data/UpholdConstants.java -d
```

Comment:
Will a testnet build use this encrypted file also?

Reply:
No, test builds use the original one.
The build script is executed line by line, so it will first do
```
- ./gradlew assemble_testNet3Debug
- rm -rf .gradle/4.4
```
then decrypt _UpholdConstants.anc_ and replace _UpholdConstants.java_
and finally build prod APKs
```
- ./gradlew assembleProdDebug
- ./gradlew assembleProdRelease
```